targets/tofino/test/p4include/tofino1_base.p4

```diff
@@ -637,7 +642,7 @@ extern DirectRegister<T> {
     T read();
 
     /// Write value to a direct register.
-    void write(in T value);
+    T write(in T value);
```

Comment:
What is this change for?

Reply:
I updated the tofino includes to the version from the latest SDE. There was a small change there.